experiments/model_c/demo.py

```diff
@@ -0,0 +1,36 @@
+"""
+This file demos on how to call api.py for TTS models.
+"""
+from api import get_device, get_tts_model, record_voice
+
+
+if __name__ == "__main__":
+
+    # inputs that can be put into argparser with proper defaults in the future
+    intput_text = "我想要吃飯，你有推薦的Restaurant嗎?"
+    speaker = "Luis Moray"
+    language = "zh-cn"
+    file_path = "./output.wav"
+    duration = 10
+    speaker_wav = "./experiments/model_c/voice_ref/chung_recorded.wav"
+
+    # retrieve model
+    device = get_device()
```

Comment:
`device` is never used or referred the follow up. Do we need this line?

Reply:
Actually, `get_tts_model` should use the device, I have merged the line as an argument of the `get_tts_model`, FYI.

Reply:
Understood. Thanks for the clarification.